templates/endpoints_v2/README.ipynb

```diff
@@ -8,35 +8,37 @@
     "\n",
     "**⏱️ Time to complete**: 10 min (20 on GCP)\n",
     "\n",
-    "This guide walks you through how to deploy optimized LLMs in Anyscale through RayLLM. It includes a number of pre-tuned configs for Llama2, Mistral, Mixtral, embedding models, and more in the `models` directory.\n",
+    "This guide walks you through how to deploy LLMs in Anyscale through RayLLM.\n",
```

Comment:
revise the time? I don't why why for GCP it says it takes longer? in both I think it takes 10 min.

Reply:
I revised the time.